src/components/Envelope.vue

```diff
@@ -103,7 +103,7 @@
 				fill-color="var(--color-primary-element)" />
 		</template>
 		<template #actions>
-			<EnvelopePrimaryActions v-if="!moreActionsOpen && !snoozeOptions">
+			<EnvelopePrimaryActions v-if="!moreActionsOpen && !snoozeOptions" id="primary-actions">
```

Comment:
```suggestion
			<EnvelopePrimaryActions v-if="!moreActionsOpen && !snoozeOptions">
```

Appears to be unused. Otherwise, please try not to duplicate the id (as multiple menus may be mounted simultaneously).